.github/scripts/rust/Makefile.toml

```diff
@@ -19,8 +19,8 @@ CARGO_RUSTDOC_FLAGS = "--all-features -Zunstable-options -Zrustdoc-scrape-exampl
 CARGO_RUSTDOC_HACK_FLAGS = "--workspace"
 CARGO_TEST_FLAGS = ""
 CARGO_TEST_HACK_FLAGS = "--workspace --feature-powerset"
-CARGO_MIRI_FLAGS = ""
-CARGO_MIRI_HACK_FLAGS = "--workspace --feature-powerset"
+CARGO_MIRI_FLAGS = "--all-features --tests --lib"
```

Comment:
It will actually run the code.
The chance it will detect problems in those branches is pretty low. Running miri is very slow so IMO this is a good trade-off.

Reply:
That's a good idea! 0cb070acd4aaee3724892e26a03a5347c3108220

Reply:
Do you want to make the Miri changes for the Engine as well? Or continue to use powerset there?

Reply:
It's applied globally as this is the top-level Makefile.

Reply:
Don't you override it in the engine makefile though?

Reply:
Seems like that's where we could benefit most from a speed-up, but fair enough 😁

Reply:
Actually, the feature powerset in error-stack is huge, especially with #747 merged 🙂